spinalcordtoolbox/scripts/sct_run_batch.py

```diff
@@ -129,8 +129,12 @@ def get_parser():
                         help='Setting for environment variable: PATH_SEGMANUAL\n'
                         'A path containing manual segmentations to be used by the script program.')
     parser.add_argument('-script-args', default='',
-                        help='A quoted string with extra flags and arguments to pass to the script. '
-                        'For example \'sct_run_batch -path-data data/ -script-args "-foo bar -baz /qux" process_data.sh \'')
+                        help='A quoted string with extra flags and arguments to pass to the script.\n'
+                             'For example \'sct_run_batch -path-data data/ -script process_data.sh '
+                             '-script-args "VARIABLE1 VARIABLE2"\'.\n'
+                             'The variables are retrieved by a script as \'${2}\', \'${3}\', etc.\n'
```

Comment:
Minor nitpick:

```suggestion
                        help='A quoted string with extra arguments to pass to the script.\n'
                             'For example \'sct_run_batch -path-data data/ -script process_data.sh '
                             '-script-args "ARG1 ARG2"\'.\n'
                             'The arguments are retrieved by a script as \'${2}\', \'${3}\', etc.\n'
```

Since we're getting rid of `-foo` and `-baz` from the description, I think including "flags" isn't accurate anymore. Plus, since it's called `-script-args`, I think standardizing on "args/arguments" rather than "variables" might be clearer?

Reply:
Good point!